build.gradle

```diff
@@ -21,7 +21,7 @@ plugins {
 }
 
 group = 'io.neonbee'
-version = '0.31.1'
+version = '0.31.2-SNAPSHOT'
```

Comment:
should it be snapshot? not sure what's the release process like in neonbee

Reply:
Normally this is done automatically, that the version is set to SNAPSHOT after NeonBee is released, but because we released NeonBee 0.31.1 "manually", I had to change this.